src/app/common/o/pipes/get-property.pipe.ts

```diff
@@ -0,0 +1,35 @@
+import { Pipe, PipeTransform } from "@angular/core";
+
+@Pipe({
+  name: "getProperty",
+})
+export class GetPropertyPipe implements PipeTransform {
```

Comment:
So just to be clear, this pipe is responsible for taking the associated genre/author/published id and sorting through them to find the genreName/authorName/publisherName associated with the given Id and displaying it?

Reply:
In practice yes, but it can be used for other things too.